spinta/nodes.py

```diff
@@ -108,6 +108,15 @@ def get_node(
             )
 
     Node_ = config.components[group][ctype]
+
+    # If it's a model type and has features, return PartialModel instead
+    if (ctype == 'model' and 
+        data is not None and 
+        'features' in data and 
+        data['features']):
+        from spinta.components import PartialModel
+        return PartialModel()
```

Comment:
Here `ctype` should be `partial_model` and `partial_model` should be added to:

https://github.com/atviriduomenys/spinta/blob/de948899400002ef57b7f3dcd95d2c3936962945/spinta/config.py#L82-L87